src/ui/components/PasswordModule/PasswordModule.tsx

```diff
@@ -44,25 +49,43 @@ const PasswordModule = forwardRef<PasswordModuleRef, PasswordModuleProps>(
       setHintValue("");
     };
 
+    const handleClearExisting = () => {
+      setAlertExistingIsOpen(false);
+      handleClearState();
+    };
+
     useImperativeHandle(ref, () => ({
       clearState: handleClearState,
     }));
 
     const handleContinue = async (skipped: boolean) => {
       if (!skipped) {
-        await SecureStorage.set(
-          KeyStoreKeys.APP_OP_PASSWORD,
-          createPasswordValue
+        const currentPassword = await SecureStorage.get(
```

Comment:
This throws an error if it doesn't exist yet - will this be an issue on onboarding?

Reply:
I had that issue @iFergal when testing on a fresh device, that's why I set that to an empty string at line 88 if the user decides to skip the password.

`await SecureStorage.set(KeyStoreKeys.APP_OP_PASSWORD, "");`

We're safe 👍